main.tf

```diff
@@ -26,7 +27,7 @@ locals {
       enabled = false
     }
     serviceAccount = {
-      create = false
+      create = true
```

Comment:
The service account should be created with a submodule

Reply:
Based on comments from @RamyaRaghuveera , this value is required to use a different service account other than default. It causes a warning in ArgoCD but we can revisit this when we upgrade the helm chart for SonarQube

Reply:
Adding the deployment.yaml snippet with create flag for future reference:
{{- if .Values.serviceAccount.create }}
    {{- if .Values.serviceAccount.name }}
      serviceAccountName: {{ .Values.serviceAccount.name }}
    {{- else }}
      serviceAccountName: {{ include "sonarqube.fullname" . }}
    {{- end }}
    {{- end }}